build/loc/Program.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Text.Json;
+using CommandLine;
+
+// ReSharper disable SuggestVarOrType_BuiltInTypes
+// ReSharper disable SuggestVarOrType_Elsewhere
+
+namespace OneLocBuildSetup
+{
+    [SuppressMessage("Style", "IDE0008:Use explicit type")]
+    internal class Program
+    {
+        private const string XlfExtension = ".xlf";
+        private const string SpanishXlfExtension = ".es" + XlfExtension;
+
+        public static void Main(string[] args) => Parser.Default.ParseArguments<Arguments>(args).WithParsed(RunSetup);
+
+        private static void RunSetup(Arguments args)
```

Comment:
>RunSetup [](start = 28, length = 8)

minor suggestion: I think having comments about what this code is doing (similar to the PR description) here in the code would be great so that we don't need to read the commit history to understand what the goal of this code is.

Reply:
Split each process into separate methods and added comments with links for what each process does.